### ISSUE TITLE
 I18nManager.forceRTL(true);

### ISSUE THREAD

**Issue by HananeAlSamrout** [leecade/react-native-swiper#314]
please fix the module in case the direction becomes from right to left using I18nManager.forceRTL(true);

**Comment by nirshubz** [comment]
+1 
@HananeAlSamrout Did you find any solution to that?
BTW, it works well on Android

**Comment by dastoori** [comment]
Swiper uses the [`ViewPagerAndroid`](https://facebook.github.io/react-native/docs/viewpagerandroid.html) on android and it supports RTL layout, but on iOS it use `ScrollView` and it has problem with RTL layout

To fix this problem, add `style={{ flexDirection: I18nManager.isRTL ? 'row-reverse' : 'row' }}`

**Comment by phpfalcon** [comment]
@dastoori thanks for sharing this but the problem is you want also to change buttons , dots .. and the wrapper flexDirection style .. I made all the changes it might help some one https://gist.github.com/phpfalcon/2939d87fc31b698c86d1f103850eec51

**Comment by anonrig** [comment]
@phpfalcon I don't think that your solution works for some reason.

**Comment by phpfalcon** [comment]
@Anoring you are right after testing the code the scrolling won't work you need to add showButtons

**Comment by phpfalcon** [comment]
@Anonrig*

**Comment by phpfalcon** [comment]
@anonrig I made some changes it appears that style not applied to the tag I've tested it now it should working fine https://gist.github.com/phpfalcon/2939d87fc31b698c86d1f103850eec51

**Comment by bilalsyed001** [comment]
https://github.com/bilalsyed001/react-native-swiper/

**Comment by developersHomefriend** [comment]
> bilalsyed001@e3a312
hi bilal,
i have implemented your fix it works fine , but when isRTl = true is android platform , the indexes are reversed (it is well displayed but indexes are reversed)

**Comment by mohmdalfaha** [comment]
Is there any solution so far ??

**Comment by barak109** [comment]
> > bilalsyed001@e3a312
> > hi bilal,
> > i have implemented your fix it works fine , but when isRTl = true is android platform , the indexes are reversed (it is well displayed but indexes are reversed)

@developersHomefriend @mohmdalfaha,  Happen to me too, any advise?

**Comment by chenop** [comment]
Created an updated fork that support RTL
Check it out here:
https://github.com/chenop/react-native-swiper

To use:
1. in package.json:
```
"dependencies": {
    ...
    "react-native-swiper": "https://github.com/chenop/react-native-swiper",
}
```
2. yarn && pod install
